apps/prairielearn/python/prairielearn.py

```diff
@@ -727,9 +728,7 @@ def get_color_attrib(element, name, *args):
             return PLColor(val).to_string(hex=True)
         else:
             raise Exception(
-                'Attribute "{:s}" must be a CSS-style RGB string: {:s}'.format(
-                    name, val
-                )
+                f'Attribute "{name:s}" must be a CSS-style RGB string: {val:s}'
```

Comment:
Isn't `:s` the default, rendering this unnecessary? Ditto elsewhere.

Reply:
`:s` will only accept a string as a value. So e.g. `{1.2:s}` would fail: https://stackoverflow.com/a/41630769. I can still remove if wanted.